### PR TITLE
fix: note about the termination of Node 6 support

### PR DESCRIPTION
**Pull request by kyegupov**
- [x] Ready for review
- [x] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/.github/CONTRIBUTING.md) rules
- [x] Reviewed by Snyk internal team

#### What does this PR do?
Upgrades the notice about Node 6 EOL to a warning, adds more details.

image

### CODE REVIEW

File: src/cli/commands/auth/index.ts [hunk 1]
```diff
@@ -79,13 +79,17 @@ async function testAuthComplete(token: string): Promise<{ res; body }> {
   };
 
   return new Promise((resolve, reject) => {
+    console.log('PL', payload);
```
Comment (FauxFaux) [inline src/cli/commands/auth/index.ts]: Um.

Comment (kyegupov) [reply src/cli/commands/auth/index.ts]: I need to remember clearing my workspaces.

**Comment by lili2311** [comment]
Please add a screenshot

File: src/cli/index.ts [hunk 2]
```diff
@@ -187,11 +187,17 @@ async function main() {
...
-          'Node.js v6 has past End-of-Life; please upgrade your current Node.js version ' +
-          'to continue scanning and protecting your projects from security vulnerabilities.',
+        msg: `== WARNING ==========================================================
+ Node.js v6 has reached end-of-life in April 2019.
+ Snyk CLI will stop supporting Node 6 after January 2020.
+ Please upgrade your Node.js (v10+ recommended) to continue scanning
+ and protecting your projects from security vulnerabilities.
+
+ More information: https://nodejs.org/en/about/releases/
+=====================================================================
```
Comment (lili2311) [inline src/cli/index.ts]: test please to show that for a legacy runtime only this is shown?

File: src/cli/index.ts [hunk 3]
```diff
@@ -1,216 +1,12 @@
 #!/usr/bin/env node
 import 'source-map-support/register';
-import * as Debug from 'debug';
```
Comment (kyegupov) [inline src/cli/index.ts]: everything was moved to `main.ts`

File: src/cli/index.ts (hunk 2 above)
Comment (kyegupov) [reply src/cli/index.ts]: Added a test for various Node versions, extracted main() to a separate module to be testable

**Comment by lili2311** [comment]
Message will come from Back End, this is no longer needed

**Pull request closed without merging**